src/components/dialogs/parameters/sensi/table-cell.tsx

```diff
@@ -37,8 +39,15 @@ function EditableTableCell(
                     name={`${arrayFormName}[${rowIndex}].${column.dataKey}`}
                     options={column.equipmentTypes}
                     disableClearable={true}
+                    allowNewValue
                     size="small"
-                    sx={{ width: getDynamicWidth(column.equipmentTypes) }}
+                    fullWidth
+                    sx={{
+                        flexDirection: 'row',
```

Comment:
you can delete this property, not a flex context

Reply:
done